src/services/formatting/smartIndenter.ts

```diff
@@ -436,7 +436,6 @@ namespace ts.formatting {
                 case SyntaxKind.Parameter:
                 case SyntaxKind.FunctionType:
                 case SyntaxKind.ConstructorType:
-                case SyntaxKind.UnionType:

```

Comment:
do I understand correctly, this is removed to avoid double indentation for cases like the one below?

```ts
type a = string
    | number;